contracts/src/v0.8/dev/keeper2_0/KeeperRegistry2_0.sol

```diff
@@ -57,10 +58,146 @@ contract KeeperRegistry2_0 is
     address linkEthFeed,
     address fastGasFeed,
     address keeperRegistryLogic,
-    Config memory config
+    OnChainConfig memory onChainConfig
   ) KeeperRegistryBase2_0(paymentModel, registryGasOverhead, link, linkEthFeed, fastGasFeed) {
     KEEPER_REGISTRY_LOGIC = keeperRegistryLogic;
-    setConfig(config);
+    setOnChainConfig(onChainConfig);
+  }
+
+  /**
+   * @inheritdoc OCR2Keeper
+   */
+  function setConfig(
+    address[] memory signers,
+    address[] memory transmitters,
+    // TODO: Also incorporate payees here
+    uint8 f,
+    uint16 numOcrInstances,
+    uint64 offchainConfigVersion,
+    bytes memory offchainConfig
+  ) external override {
+    // Executed through logic contract
+    _fallback();
+  }
+
+  /**
+   * @inheritdoc OCR2Keeper
+   */
+  function latestConfigDetails()
+    external
+    view
+    override
+    returns (
+      uint32 configCount,
+      uint32 blockNumber,
+      bytes32 rootConfigDigest,
+      bytes32[] memory configDigests
+    )
+  {
+    uint16 numOcrInstances = s_numOcrInstances;
+    rootConfigDigest = s_latestRootConfigDigest;
+    configDigests = new bytes32[](numOcrInstances);
+    for (uint256 i = 0; i < configDigests.length; i++) {
+      configDigests[i] = rootConfigDigest ^ bytes32(i);
+    }
```

Comment:
open q. Do we need to return multiple config digests or can off chain system compute by itself.
cc @begmaroman